src/map/LayerHelper.js

```diff
@@ -62,7 +62,7 @@ export function LayerHelper($q, $http, ngeoTilesPreloadingLimit) {
    * The Tiles Preloading Limit value
    * @type {number}
    */
-  this.tilesPreloadingLimit_ = ngeoTilesPreloadingLimit ? ngeoTilesPreloadingLimit : Infinity;
+  this.tilesPreloadingLimit_ = ngeoTilesPreloadingLimit === null ? ngeoTilesPreloadingLimit : Infinity;
```

Comment:
Mhhh, so if the value is null, we use null, otherwise Infinity ? It is not better with:
```suggestion
  this.tilesPreloadingLimit_ = ngeoTilesPreloadingLimit === null ? Infinity : ngeoTilesPreloadingLimit;
```

Reply:
+1 Thanks :-)